src/Features/Core/Portable/Shared/Extensions/ISymbolExtensions_2.cs

```diff
@@ -153,6 +153,10 @@ MethodKind.Conversion or
                 case SymbolKind.TypeParameter:
                     return Glyph.TypeParameter;
 
+                // Temporary fix
```

Comment:
temp fix for what?

Reply:
Added an existing glyph so that the impl runs. In the future, a special glyph for preprocessing symobls could be needed.

Reply:
what do you mean "so that the impl runs"?

Reply:
we should use whatever glyph we are already using today in completion for PP directives.

Reply:
Today we are using the keyword glyph for both preprocessor directives and defined preprocessing symbols. Using that makes more sense in this context.